canonical-web-and-design/maas-ui
# Add LP to GH issues script and cron

## Done
- Created a scripts directory to hold the bug mitigator script
- Wrote a script that gets all LP bugs tagged "ui" which does not have a linked bug tracker with maas-ui
- Add an action to run the script overnight

## QA
Check the code. The action is likely to fail due to a lack of an LP bot account which I will request.

_If the bot takes a few days, I will disable the action but I want to see it run once to check the action itself_

## Fixes

Fixes: https://github.com/canonical-web-and-design/MAAS-squad/issues/2047.
